src/api/v1beta1/properties.go

```diff
@@ -389,10 +389,10 @@ func (dk *DynaKube) DefaultOneAgentImage() string {
 		return ""
 	}
 
-	tag := api.LatestTag
+	tag := api.RawTag
 	if version := dk.CustomOneAgentVersion(); version != "" {
 		truncatedVersion := truncateBuildDate(version)
-		tag = truncatedVersion
+		tag = truncatedVersion + "-" + api.RawTag
```

Comment:
```suggestion
		tag += "-" + truncatedVersion
```
the `tag` variable already has a value of `raw`

Reply:
yes, but the tag should be "1.234.0-raw", not "raw-1.234.0". One could write `tag = truncatedVersion + "-" + tag` but that's more confusing than using the raw tag a second time.

Reply:
I prefer having the version at the end, since it's the optional part.

we would always have raw at the start and then an optional version after it

Reply:
I didn't realize that both tags are actually supported by the tenant registry.